modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -110,7 +110,8 @@ export default class I3SConverter {
 
     const fetchOptions = await this._fetchPreloadOptions();
     const sourceTilesetJson = await load(inputUrl, CesiumIonLoader, {
-      fetch: fetchOptions
+      fetch: {headers: fetchOptions.headers},
```

Comment:
Is it your intention to discard all other fetchOptions and just pass in headers?
```suggestion
      fetch: fetchOptions,
```